### PR TITLE
Portfolio setup mobile first pull request 

### PR DESCRIPTION
**Pull request by tamoorsaeed22**
Summary 
1:Used flex for the session.
2:Used microverse template 1 for reference.
3:FIGMA design has been used to make this portfolio as per requirement.
4:Linting tool that helps improve the site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
5:Followed GitHub flow.
6:Create branches with different base branches
7:In this project, I have set up a portfolio. My goal was to use all of the tools and practices.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      <a href="#" class="logo">My logo</a>
+  <img src="Images/Union.png" class="nav-bar">
+      </nav>
+      </header>
+     <main class="container">
+        <div class="background">
+    <section class="home-section">
+
+      <h1 class="header">
+        I'm Tamoor<br>
```
Comment (osamaashraf6) [inline index.html:22]: - [ ] Great job and have a nice design 😃, but I see that the content of the home section wants to go up a little to be like the design the margin between the header and the content bigger than the Figma Design, so kindly 😊 try to fix the margin between them to be like the Figma design and the design will be attractive for the visitors 👌

> Check the table below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image  |    image|

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        Glad to see <br>
+        you!
+    </h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product ,feature or website Look through some of my work and experience! If you like what you see and have a project you need coded , 
+        don’t hestiate to contact me.
+    </p>
+<a href="#" class="link">LET'S CONNECT</a>
```
Comment (osamaashraf6) [inline index.html:30]: - [ ] I see 😃 that you put a big value on the margin between elements, so kindly try to fix that and use the correct margin to be like the Figma Design👌

> Check the table below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|    image  |    image|

File: index.html (hunk 1 above)
Comment (osamaashraf6) [inline index.html:22]: - [ ] I see that the type of the font family is not like the Figma so, Kindly 😃try to use the correct font family to be like the Figma Design 💯

> Check the table below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image |   image |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @tamoorsaeed22 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️

- [x] Good commit messages.
- [x] Descriptive READme file in a professional way.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow
- [x] Nice design.




## Required Changes ♻️

- [ ] kindly👇💯 check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by tamoorsaeed22** [comment]
1: content bigger than the Figma Design has been resolved.
2:Margin between Header and text has been resolved.
3:font family has been changed according to Figma design .

**Review by Ibizugbe — APPROVED** [review]
Hi @tamoorsaeed22 👋, 

BigTvGIF (2)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅
- github flow has been used👍
- linters are passing.👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**